ui/pages/confirmations/components/confirm-page-container/confirm-page-container.component.js

```diff
@@ -213,7 +214,12 @@ const ConfirmPageContainer = (props) => {
           />
         ) : (
           <ConfirmPageContainerHeader
-            showEdit={showEdit}
+            showEdit={
+              origin === ORIGIN_METAMASK ||
+              currentTransaction.type === TransactionType.simpleSend
```

Comment:
If we wanted to continue allowing token transfer edits:
```suggestion
              [TransactionType.simpleSend, TransactionType.tokenMethodTransfer].includes(currentTransaction.type)
```